ImGuiDebugPanel.cpp

```diff
@@ -1,3 +1,4 @@
+#include <cmath>
 #include "TextEditor.h"
```

Comment:
where are we using sqrt in this file? i have the same question regarding `LanguageDefinitions.cpp` and `UnitTests.cpp`

Reply:
It is used inside TextEditor.h

Reply:
why not add the include there then?

Reply:
Yes.

Reply:
Done